include/trieste/wf.h

```diff
@@ -81,10 +81,26 @@ namespace trieste
           tokens.end(),
           std::back_inserter(offsets),
           [&](const Token& t) {
-            return 1.0 /
-              (1.0 +
-               (alpha * (depth - target_depth) *
-                token_terminal_distance.at(t)));
+            
+            if (token_terminal_distance.find(t) != token_terminal_distance.end())
```

Comment:
Was this necessary for C++17, or just helpful for debugging.

Reply:
I added it in whilst debugging, and decided to keep it as it is otherwise a hard thing to diagnose.